kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -340,6 +343,14 @@
     )
     private int kopPrometheusStatsLatencyRolloverSeconds = 60;
 
+    @FieldContext(
+            category = CATEGORY_KOP,
+            doc = "The allowed namespaces to list topics with a comma separator.\n"
+                    + " For example, \"public/default,public/kafka\".\n"
+                    + "If it's not set or empty, the allowed namespaces will be \"<kafkaTenant>/<kafkaNamespace>\"."
+    )
+    private String kopAllowedNamespaces;
```

Comment:
We should check if the provided namespace is valid to void any invalid namespace names, In Pulsar `NamespaceName.validateNamespaceName()` can be used to check.

Reply:
And for the configuration class, I think we can define as Set<String> directly. Link to https://github.com/apache/pulsar/blob/6704f12104219611164aa2bb5bbdfc929613f1bf/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L914